doto-frontend/src/components/pages/Settings/SettingsPage.js

```diff
@@ -0,0 +1,124 @@
+import { AccountCircle } from "@material-ui/icons";
+import EmailIcon from "@material-ui/icons/Email";
+import CloudUploadIcon from "@material-ui/icons/CloudUpload";
+import SaveIcon from "@material-ui/icons/Save";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core/styles";
+import DateFnsUtils from "@date-io/date-fns";
+import { MuiPickersUtilsProvider, KeyboardTimePicker } from "@material-ui/pickers";
+
+const theme1 = createMuiTheme({
+    palette: {
+        primary: {
+            main: "#3700B3",
+        },
+        secondary: {
+            main: "#7cb342",
+        },
+    },
+});
+
+const SettingsPage = () => {
+    const [selectedStartTime, setSelectedStartTime] = React.useState(new Date("2020-03-15T09:00:00"));
+    const [selectedEndTime, setSelectedEndTime] = React.useState(new Date("2020-03-15T17:00:00"));
+
+    const handleStartTimeChange = date => {
+        setSelectedStartTime(date);
+    };
+
+    const handleEndTimeChange = date => {
+        setSelectedEndTime(date);
+    };
+
+    return (
+        <div className="SettingsPage">
+            <div className="left-side-bar" />
+            <div className="right-side-bar">
+                <FormControl id="input-field">
+                    <InputLabel>Name</InputLabel>
+                    <Input
+                        startAdornment={
+                            <InputAdornment position="start">
+                                <AccountCircle />
+                            </InputAdornment>
+                        }
+                    />
+                </FormControl>
+
+                <FormControl id="input-field">
+                    <InputLabel>Email</InputLabel>
+                    <Input
+                        startAdornment={
+                            <InputAdornment position="start">
+                                <EmailIcon />
+                            </InputAdornment>
+                        }
+                    />
+                </FormControl>
+
+                <div className="column-fields">
+                    <Avatar id="profile-photo" alt="John Smith" src="/upload.jpg" variant="square" />
+                    <Button id="upload-photo" variant="contained" startIcon={<CloudUploadIcon />}>
+                        Upload Photo
+                    </Button>
+                </div>
+
+                <div className="column-fields">
+                    <h2 style={{ marginLeft: "10vw", marginTop: "4vh", textAlign: "left" }}>Theme:</h2>
+                    <div>
+                        <ThemeProvider theme={theme1}>
+                            <Button id="color-palette" variant="contained" color="primary"></Button>
+                        </ThemeProvider>
+                        <ThemeProvider theme={theme1}>
+                            <Button id="color-palette" variant="contained" color="secondary"></Button>
+                        </ThemeProvider>
+                    </div>
```

Comment:
are the empty div's really necessary? line 71 and 78

Reply:
No. Just took them out.